src/components/VTable/VTable.vue

```diff
@@ -1,138 +1,112 @@
 <template>
-  <div>
-    <!-- <div className="lists-container">
-      <h2 v-if="caption">
+  <div
+    ref="container"
+    class="table-container"
+    tabindex="0"
+    role="group"
+    aria-labelledby="caption"
+  >
+    <table>
+      <caption
+        v-if="caption"
+        id="caption"
```

Comment:
An ID should only occur once on a page. if someone uses multiple `<vtable>`'s on a page there will be multiple `#caption` ID's. This should be set to something more dynamic.

Reply:
Yep. Totally right. And in fact, I already fixed it in my `develop` branch. Should have mentioned to you that I work off that usually, but I wasnt expecting you to stick around so long :)

Also open to any suggestions on Git strategies. Currently I like to keep master as a reflection of what is on NPM and do any dev work off `develop`

Reply:
For my projects I keep master as the working branch and doing manual releases when I feel like it. All other code will be done in a PR and not merged to master until it works and is polished. But I don't really care. whatever works for you. But you should probably change the default branch to dev if you want people to branch off of that.

Reply:
Interesting. Yeah, I think I'll reconcile the develop branch into master for the next release, then just work off PRs from master. Will make more sense to people I think